go.mod

```diff
@@ -32,6 +32,8 @@ require (
 )
 
 require (
+	github.com/kyma-project/template-operator v0.0.0-20230925081606-4ae25406be03
+	github.com/pkg/errors v0.9.1
```

Comment:
also please don't introduce this errors as direct dependencies, it's a deprecated library since go lang natively introduced fmt.Errorf